packages/typespec-vscode/src/vscode-cmd/emit-code/emit-code.ts

```diff
@@ -501,6 +496,15 @@ export async function emitCode(context: vscode.ExtensionContext, uri: vscode.Uri
   };
   /* display existing emitters in config.yaml. */
   if (existingEmitters && existingEmitters.length > 0) {
+    const recentlyUsedSeparator = {
+      label: "recently used",
```

Comment:
how about "from tspconfig.yaml"? which is more accurate and easier to understand.

Reply:
updated